pykg2tbl/__init__.py

```diff
@@ -11,3 +11,7 @@
 import logging
 
 log = logging.getLogger(__name__)
+
+from pykg2tbl.j2.jinja_sparql_builder import J2SparqlBuilder
+from pykg2tbl.kg2tbl import KGSource
+from pykg2tbl.query import QueryResult
```

Comment:
you probably get flake8 errors warning that these imports are never used?

you get rid of them by providing an __all__ variable that lists what gets imported with a *
also I would like us to hide the J2SparqlBuilder behind a DefaultSparqlBuilder in the "public interface"

```python
# warning: untested code fragment
DefaultSparqlBuilder = J2SparqlBuilder  # just in case this ever changes to say a J3SparqlBuilder...

__all__ = ["DefaultSparqBuilder", "KGSsource", "QueryResult"]
```

see also https://discuss.python.org/t/usage-of-all-in-init-py/17936/4